azkaban-web-server/src/test/java/azkaban/webapp/servlet/ProjectPageHeaderTest.java

```diff
@@ -19,7 +19,7 @@ public class ProjectPageHeaderTest {
   @Test
   public void testUploadButtonIsPresent() {
     final VelocityContext context = VelocityContextTestUtil.getInstance();
-    context.put("hideUploadProject", false);
+    context.put("projectUploadLock", false);
```

Comment:
For these 2 changes on the test case, I suggest we keep the one tests, while copying and create new, if the old `hideUploadProject` key is still in use.

Reply:
no, the old will not work because I overwrite it. This hideUploadProject value depends on projectUploadLock